docs/reference/schema-viewer-navigation.md

```diff
@@ -0,0 +1,179 @@
+    followRedirects = true
+    readTimeoutMs=200000
+  } # second key-value pair
+}
+
+...
+
+```
+It is worth noting that the `WebserviceFileDataObject` element appears twice, each time with distinct keys. Initially, it is employed for downloading departure dates, and subsequently for acquiring airport information data.
+
+In contrast to `[mapOf]` elements, elements marked by `[oneOf]` can only accommodate a single nested element. These elements function as a specialized type of property, demanding the inclusion of one specific sub-element.
+
+Example from schema browser (schema{}->actions[mapOf]*->CopyAction{}->executionMode[oneOf]):
+
+:::info 
+There is a special property called (const). It is found on the type property of an object.  This particular property is required to contain the class name of the parent element e.g. CopyAction. 
+:::
+```
+actions {
+  select-airport-cols {
+    type = CopyAction # special property of type (const)
+    executionMode = { type = DataObjectStateIncrementalMode } # [oneOf] element executionMode containing the nested element of type DataObjectStateIncrementalMode
+    ...
+  }
+}
+
+...
+
+```
+
+
+Arrays or lists use the square brackets notation `[]` in an elements name. They are used for example in the transformer element for creating transformation pipelines that are processed sequentially. 
+
+Example from schema browser (schema{}->actions[mapOf]*->CustomDataFrameAction{}->transformers[]):
+```
+actions {
+  join-departures-airports {
+    type = CustomDataFrameAction
+    inputIds = [int-departures, int-airports]
+    outputIds = [btl-departures-arrivals-airports]
+    transformers = [{
+      type = SQLDfsTransformer
+      code = {
+        btl-connected-airports = """
+          select int_departures.estdepartureairport, int_departures.estarrivalairport,
+            airports.*
+          from int_departures join int_airports airports on int_departures.estArrivalAirport = airports.ident
+        """
+       }
+      },
+      {
+        type = SQLDfsTransformer
+        code = {
+          btl-departures-arrivals-airports = """
+            select btl_connected_airports.estdepartureairport, btl_connected_airports.estarrivalairport,
+              btl_connected_airports.name as arr_name, btl_connected_airports.latitude_deg as arr_latitude_deg, btl_connected_airports.longitude_deg as arr_longitude_deg,
+              airports.name as dep_name, airports.latitude_deg as dep_latitude_deg, airports.longitude_deg as dep_longitude_deg
+            from btl_connected_airports join int_airports airports on btl_connected_airports.estdepartureairport = airports.ident
+          """
+        }
+      }
+    ] # transformer array element containing two SQLDfsTransformer elements
+
+    ...
+
+
+  }
+}
+
+...
+
+```
+
+
+
+Finally, elements ending with `(type)`  denote properties that contain values of a predetermined type, such as string, integer, or other specified data types.
```

Comment:
This confuses a little with the `type` property. I suggest to highlight a little different an use a different example description below. How about:
Finally, elements ending with a *type* in brackets, e.g. `persist(boolean)` denote properties that contain values of a predetermined *type*, such as `string`, `integer`, `boolean` or other specified data types.